### DISCUSSION TITLE
When will you support Whatsapp Business

### DISCUSSION THREAD

**Discussion by alssadah** [botman/botman#1328] [Ideas]
Whatsapp Cloud API is now available, so why don't you provide a package that we can use.

**Comment by filippotoso** [comment]
When someone in the community will implement a driver for it.

**Comment by divyaniverma** [comment]
WhatsApp Business is a messaging app specifically designed for businesses to communicate with their customers.It offers a verified Business Profile to enable greater trust in messaging support, which is suitable for managing lower volumes of customer requests for smaller businesses. Meanwhile, larger businesses with an enterprise, commercial.

Most businesses can use WhatsApp Business API to integrate the app into CRM and customer support software for greater scalability. WhatsApp Business offers many features that can benefit small businesses, including verified business profiles, quick replies, automated greetings, message statistics, interactive messages, integration with other platforms and more. Read More msgclub.net

**Comment by filippotoso** [comment]
Feel free to implement the driver. Here's the botman documentation: https://botman.io/2.0/drivers
